docker/secondary-analysis-python/Dockerfile

```diff
@@ -12,4 +12,4 @@ RUN mkdir /tools
 
 COPY tools /tools
 
-RUN pip install -e '/tools/submit'
+RUN pip install -e '/tools/secondary_analysis'
```

Comment:
I'm wondering why we made the decision to install in develop mode instead of doing a regular `pip install setup.py`. Was it to avoid using a `MANIFEST.in` to copy over non-python files?

Reply:
Do you mean instead of doing `python setup.py install`? I think we ended up doing it this way simply because I was trying to use pip to install it and `pip install -e dir` is the way the interwebs tell you to do it: https://stackoverflow.com/questions/15031694/installing-python-packages-from-local-file-system-folder-with-pip

Is there an advantage to doing it another way, like `python setup.py install` instead?

Reply:
The biggest functional difference that I've seen is that `-e` or develop mode soft-links to the directory that you're installing from: changes to that directory will be reflected in your code. 

`python setup.py install` installs into your `site-packages directory` (and leaves behind anything not in `MANIFEST.in`. 

Here I doubt there is a, but `python setup.py install` is a "normal" way to install the package, and is a bit more strict about how the package contents is defined in `setup.py`. If we go with this way and everything works, there are slightly better guarantees of portability.

Reply:
Okay, changing to `python setup.py install` sounds like a worthwhile change. @samanehsan do you think you could experiment with that and see if you can put it in this PR? If it turns out to be a lot of work we should put it in a separate ticket and PR.